Flow.Launcher/Languages/it.xaml

```diff
@@ -80,6 +83,7 @@
     <system:String x:Key="searchplugin_Noresult_Title">No results found</system:String>
     <system:String x:Key="searchplugin_Noresult_Subtitle">Please try a different search.</system:String>
     <system:String x:Key="plugin">Plugin</system:String>
+    <system:String x:Key="plugins">Plugins</system:String>

```

Comment:
Just update `en.xaml` should be fine. This is fairly annoying to update all files

Reply:
You will not understand this. People with low IQs usually work like this.

Reply:
Na I just want to inform you the fact so next time you don't have to do it🥺

Reply:
If I follow you, doesn't every other language display "plugins"?.
I don't want to affect other languages, so I copied all the 'plugin' values into the 'plugins'.
If I didn't want to influence other languages, isn't this the right way to do it?

Reply:
The issue is crowdin won't automatically load translation....and I think it's ok to let translator revises the string if we change the source string. Sometimes crowdin can detect the change from `plugin->plugins` and will revise the translation in other language as well.

Reply:
No no need for that. I am just saying it won't be flushed into crowdin